### DISCUSSION TITLE
React hook crashes if form copies have a radio button.

### DISCUSSION THREAD

**Discussion by bbg** [react-hook-form/react-hook-form#8063] [General] [answered]
Hi everyone,

Application crash when there are too many react-hook-form based forms and radio buttons in them.
To see the error; Press the OpenForm button. Click any radio button to select then try to type something in the content(textarea) field you will see the application crashed.
The problem is solved when I remove the radio buttons. I can't understand why I am having this problem.

https://codesandbox.io/s/flow-ckhobm

Thank you

**Comment by Moshyfawn** [comment, marked as answer]
It looks like the issue is nothing to do with RHF, but with your UUID generator function. If you pass a stable value _(at least, the premiseType index, tho, not recommended)_ as your label `key` prop, the issue disappears.

**Reply by bbg** [reply]
@Moshyfawn Very interesting, I never thought of it, thank you.
